soc/riscv/riscv-ite/it8xxx2/Kconfig.soc

```diff
@@ -59,6 +59,24 @@ config SOC_IT8XXX2_GPIO_H7_DEFAULT_OUTPUT_LOW
 	  capability to pull it down. We can only set it as output low,
 	  so we enable output low for it at initialization to prevent leakage.
 
+config SOC_USBPD_ITE_PHY_PORT_COUNT
+	int "Number of ITE USBPD physical port counts"
+	default 2
+	help
+	  ITE EC chip has embedded integrated pd module, this is different
+	  from standalone TCPC. Both it81202 and it81302 support two usbpd
+	  ports on chip.
```

Comment:
"This defines the total number of USBPD ports available on the ITE EC chip.  Both it81202 and it81302 support two USBPD ports."

Reply:
Done